spell/spell-crafter-mainnet-workflow.md

```diff
@@ -146,7 +155,24 @@ Repo: https://github.com/makerdao/spells-mainnet
 * [ ] Wait for CI to PASS
 * [ ] Wait for at Least two Approvals to Share for Publishing to GovAlpha
 * [ ] Share Deployed Address in `new-spells`
-* [ ] Fill Spell Crafter Related Boxes in the Main Exec Doc Sheet
+* [ ] Fill Spell Crafter Related Boxes in the Main Exec Sheet
```

Comment:
Could we move this to the earlier point in the process? New governance team has requested this to be moved to an earlier point in the checklist so that they have confirmations earlier. 

I would suggest moving to the beginning of the list goerli crafter checklist before `Add Spell Actions as per [Exec Sheet]`